bescort.lic

```diff
@@ -584,6 +590,64 @@ class Bescort
       end
     end
   end
+  
+  def take_theren_rope_bridge(mode)
+	theren_side_roomid = 8650
+	rossman_side_roomid = 8637
+	@target_room = false
+	@shuffle_direction = false
+    if mode == 'totheren'
+		if not Room.current.id == rossman_side_roomid
+			echo("Must start on the south side of the rope bridge")
+			exit
+		end
+		@shuffle_direction = "north"
+		@target_room = theren_side_roomid
+	elsif mode == 'torossman'
+		if not Room.current.id == theren_side_roomid
+			echo("Must start on the north side of the rope bridge")
+			exit
+		end
+		@shuffle_direction = "south"
+		@target_room = rossman_side_roomid
+	else
+		echo("Unrecognized argument: #{mode}")
+		exit
+	end
+	
+	start_room = Room.current.id
+	Flags.add("rope_wait","finally arriving on this side","finally reaching the far side")
+	while Room.current.id == start_room
+		put("retre") # RT irrelevant
+		put("retre")
+		Flags.reset("rope_wait") # avoiding race conditions
+		ret = bput("climb rope", "Roundtime", "is already on the rope", "You climb onto a", "You are engaged")
```

Comment:
I know that's present in a lot of places in dr-scripts, but I've gotten in the habit of not doing that since I often use more advanced string operations on the result than just === or nest logic on it. I also usually don't bother when I only check one thing. Granted, in this case, I may as well case it.